apollo-federation/src/query_plan/mod.rs

```diff
@@ -235,11 +235,13 @@ pub struct FetchDataKeyRenamer {
 /// elements.
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
 pub enum FetchDataPathElement {
-    Key(Name),
-    AnyIndex,
+    Key(Conditions, Name),
```

Comment:
should this be flipped to match the `json_ext::PathElement` from `apollo-router` crate?

Reply:
That works too, fixed in beb595dc8